.circleci/config.yml

```diff
@@ -75,7 +75,7 @@ jobs:
     environment:
       - NODE_ENV: test
     steps:
-      - run: yarn test:jest:v1 -w 2
+      - run: yarn test:ci:jest --v1 -w 2
```

Comment:
This is a hypothetical API for this command -- being able to pass in a bash arg to point at what needs to run